src/store/models/network.ts

```diff
@@ -508,6 +515,36 @@ const networkModel: NetworkModel = {
       getStoreActions().designer.updateBackendLink({ lnName, backendName });
     },
   ),
+  updateTaroBackendNode: thunk(
+    async (
+      actions,
+      { id, taroName, LNDName },
+      { injections, getState, getStoreActions },
+    ) => {
+      const networks = getState().networks;
+      const network = networks.find(n => n.id === id);
+      if (!network) throw new Error(l('networkByIdErr', { networkId: id }));
+      const lndNode = network.nodes.lightning.find(n => n.name === LNDName);
+      if (!lndNode) throw new Error(l('nodeByNameErr', { name: LNDName }));
+      const taroNode = network.nodes.taro.find(n => n.name === taroName) as TarodNode;
+      if (!taroNode) throw new Error(l('nodeByNameErr', { name: taroName }));
+      if (taroNode.lndName === LNDName)
+        throw new Error(l('connectedErr', { lnName: taroName, backendName: LNDName }));
+
+      taroNode.lndName = lndNode.name;
+      // update the network in the redux state and save to disk
+      actions.setNetworks([...networks]);
+      await actions.save();
+      // save the updated compose file
+      await injections.dockerService.saveComposeFile(network);
+
+      if (network.status === Status.Stopped) {
```

Comment:
It doesn't look like you need this check here since this action is never called when the network isn't stopped.